body.tex

```diff
@@ -231,6 +231,43 @@ \subsection{Epic-Based Long Term Plans}
 agile way, while also ensuring that scope, cost and schedule are
 carefully controlled.
 
+
+\subsection{Defining the Schedule with Milestones}
+
+Rubin Milestones are defined as Jira issues of type ``Milestone''. As indicated above, the Director (or their designate) defines the L1 milestones, the Associate Directors (ADs) define their departments' L2 milestones, and the Team Leaders and ADs define the L3 milestones for their teams.
+
+The following fields must be filled out when defining a new milestone:
```

Comment:
Once Cathy has created the milestone issues in Jira, then the ADs and Team leads go and the full description.